tests/helpers/locksAreDeployed.js

```diff
@@ -17,6 +17,15 @@ function resolveWhenDeployed(address, delay, maxAttempts) {
   return new Promise((resolve, reject) => {
     function retrieveCode() {
       attempts += 1
+
+      const retryOrReject = () => {
```

Comment:
It may be worthwhile for us to tighten up this method in the future, I could see future us tracing this to figure out how it is supposed to work